server/secure_test.go

```diff
@@ -0,0 +1,395 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/theplant/appkit/log"
+)
+
+//////////////////////////////////////////////////
+// CSRF Required Header Verification
+
+func ExampleCSRFHeader_Blank() {
+	cfg := CrossSiteConfig{
+		RawAllowedOrigins:  "http://example.com",
+		CSRFRequiredHeader: "", // Explicitly blank
```

Comment:
In real projects, does there any case that we need to set `CSRFRequiredHeader` to blank?

From the definition, it seems we can't set the `CSRFRequiredHeader` to a blank value using `jinzhu/configor`.

https://github.com/theplant/appkit/pull/14/files#diff-d3c7c3dd6a847fadf23d3fa4dd0b551dR48

Reply:
Yes, any app rendering HTML templates, `POST`ing back to the server and using CSRF tokens (e.g. https://github.com/justinas/nosurf) would need to set this to blank, since the header won't be set by the browser.

> we can't set the CSRFRequiredHeader to a blank value using jinzhu/configor.

$*(#(*$(*#$( Configor! Is there a way to set it to blank via YAML?

Reply:
>  Is there a way to set it to blank via YAML?

(Just confirmed locally) No. 

![image](https://user-images.githubusercontent.com/349931/41077232-0647ce1a-6a48-11e8-9c1a-0e0b0dda7027.png)

Reply:
Right: https://github.com/jinzhu/configor/blob/ec34f328956f53dfbd06e9f5306e37e6943443c3/utils.go#L206-L209